### DISCUSSION TITLE
Improve README on editing PR

### DISCUSSION THREAD

**Discussion by Ryuno-Ki** [mdn/mdn-community#552] [Content]
Today, several contributors asked how to edit files on a PR.

> For this, you will go to the files tab of the PR, pick the file you want, click the three dots menu in the upper right corner and go to Edit File. The changes should result in a new commit.

Are you okay with me improving the README with this instruction?

**Comment by teoli2003** [comment]
Consider: you will go -> go    and    should -> will

(I'm not a native speaker so it may be better your way)

**Reply by Ryuno-Ki** [reply]
I'm not a native speaker either.
The above was more like a suggestion on the direction I want to go.
The exact phrasing will be debated in the PR itself, I guess.

**Reply by teoli2003** [reply]
Sounds good to me!

**Reply by Ryuno-Ki** [reply]
Great! Okay, will cook a PR then.

**Comment by Rumyra** [comment]
Feel free to assign me as a reviewer on the PR @Ryuno-Ki 👍

**Reply by Ryuno-Ki** [reply]
Will do. It's a bit busy for me, but I have this on my agenda.

**Reply by Ryuno-Ki** [reply]
Finally got around to fill #7563

**Comment by pepelsbey** [comment]
Closing as answered/settled https://github.com/mdn/content/pull/7563
